serverless/serverless-getting-started.adoc

```diff
@@ -13,23 +13,32 @@ include::modules/technology-preview.adoc[leveloffset=+1]
+// [id="serverless-ha_{context}"]
+// == High availability in {ServerlessProductName}
+
+// Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
+// HA functionality is available in {ServerlessProductName} for the Knative Serving control plane.
+
+// In a HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
 
-[id="knative-cli_{context}"]
-== Knative CLI
-The Knative CLI (`kn`) extends the functionality of the `oc` or `kubectl` tools to enable interaction with Knative components on {product-title}. `kn` allows developers to deploy and manage applications without editing YAML files directly.
+// Active/passive HA in {ServerlessProductName} is available through leader election.
+// When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required.
 
-[id="applications_{context}"]
-== Applications on {ServerlessProductName}
-Applications are created using Custom Resource Definitions (CRDs) and associated controllers in Kubernetes, and are packaged as OCI compliant Linux containers that can be run anywhere.
+// These controller instances compete to use a shared resource, known as the leader election lock.
+// The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
 
-To deploy applications in {ServerlessProductName}, you must create Knative Services. For more information see xref:../serverless/knative_serving/getting-started-knative-services.adoc#getting-started-knative-services[Getting started with Knative Services].
+// Leader election is enabled for the Knative Serving control plane by default when a KnativeServing custom resource definition (CRD) is created during installation.
+
+// TODO: Add autoscaling mention; check in with PM on other topics
+
+[id="knative-eventing_{context}"]
+== Knative Eventing
+A developer preview version of Knative Eventing is available for use with {ServerlessProductName}. However, this is not included in the {ServerlessOperatorName} and is not currently supported as part of this Technology Preview. For more information about Knative Eventing, including installation instructions and samples, see the link:https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on {product-title}] documentation.
```

Comment:
Is this whole paragraph there just for one sprint? When we go GA the Knative Eventing will be Tech. Preview.

Reply:
It was there previously (has been for multiple releases, but I moved stuff around and that's why it's showing up as new. You can ignore it, I'll be removing it for GA as per the comment below it.